tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_homescreen_change_wallpaper.py

```diff
@@ -18,13 +18,14 @@ def test_homescreen_change_wallpaper_from_gallery(self):
         """
         https://moztrap.mozilla.org/manage/case/1902/
         """
-
-        homescreen = Homescreen(self.marionette)
         self.apps.switch_to_displayed_app()
-
         default_wallpaper_settings = self.data_layer.get_setting('wallpaper.image')
-        contextmenu = homescreen.open_context_menu()
-        activities = contextmenu.tap_change_wallpaper()
+
+        # open settings app and choose to change wallpaper, instead of opening context menu
+        settings = Settings(self.marionette)
+        settings.launch()
+        homescreen_page = settings.open_homescreen()
+        activities = homescreen_page.pick_wallpaper()
```

Comment:
I just realized we have the same test at `gaia/tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_wallpaper.py`. We can remove this test, then